packages/design-system-docs/src/pages/components/Button/button.example.html

```diff
@@ -1,4 +1,5 @@
 <h6 class="preview__label">Default button</h6>
+<p><code>.ds-c-button</code></p>
```

Comment:
Placing the class names for each button group does make it easier for coders to reference, but the layout looks nicer without it. Maybe remove the class names code since people can view the code when they expand the `Code snippet`

Reply:
I've always found it more difficult to look through the code blocks because there is often so much in there. We pull the classes out for all the utility pages and I was thinking it would be more helpful to see the class next to the visual. 

I think this would be something to ask folks to see which version they prefer?

Reply:
Getting a lot of feedback in Slack and folks so far overwhelmingly prefer to see the classname inline next to the element.

Reply:
https://adhoc.slack.com/archives/C201U8U7N/p1622038453066200 
https://adhoc.slack.com/archives/CL3CQ9LPR/p1622038580058400

Reply:
Going to leave the classes inline based on feedback from the polls in Slack

Reply:
Yes, go with that which has better feedback, thanks for doing the extra polling.